DataFormats/Detectors/PHOS/include/DataFormatsPHOS/Cluster.h

```diff
@@ -21,6 +21,19 @@ class Geometry;
 /// \class Cluster
 /// \brief Contains PHOS cluster parameters
 
+struct CluElement {
+  short absId;
+  bool isHG;
+  int label;
+  float energy;
+  float time;
+  float localX;
+  float localZ;
+  float fraction;
```

Comment:
if you rely on default c-tor, better provide default init. values of data members, e.g. `short absId = 0;`

Reply:
added def values